reactive/cwr.py

```diff
@@ -107,6 +107,12 @@ def install_jenkins_jobs(connected_jenkins):
         if not installed:
             hookenv.log("installation of {} did not complete on time."
                         .format(plugin))
+            # Retrying does not play well here.
+            # I have seen a case where Jenkins mistakenly reports that
+            # the plugin is installed causing an infinite retry loop :(
+            # Here we report that jenkins failed and we block.
+            set_state("jenkins.jobs.failed")
```

Comment:
Does this state ever need to be removed?  Or is the only option to remove / readd the cwr relation, thereby resetting all cwr states?